config.yaml

```diff
@@ -17,21 +27,16 @@ common:
    --plugin-option=tmpfs:keep_mounted=True
    --plugin-option=tmpfs:max_fs_size=32g
    --plugin-option=tmpfs:required_ram_mb=39800"
- commit_updates: True
- push_updates: True
-build_packages:
- packages:
- packages_repos_target_path: "/var/lib/host-os/repositories"
- keep_build_dir: False
- result_dir:  "./result"
- force_rebuild: False
 build_release_notes:
  repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  repo_branch: "master"
+ push_repo_url: "git@github.com:open-power-host-os/open-power-host-os.github.io.git"
```

Comment:
We always push to the main repository through PRs, not directly. We may want to push directly automatically in our weekly builds in the future, but that can be explicitly set in the weekly job.

Someone may be testing the commands and not realize the default is to push to the main repository. Forcing to use `--push-repo-url` makes the user think (or, at least, look) about where he wants to push.